### ISSUE TITLE
Adding #include <linux/vmalloc.h> to file and result explanation.

### ISSUE THREAD

**Issue by ghost** [andre-richter/pcie-lat#1]
The code was not working for my server, running on Ubuntu 14.04 server. Please include the vmalloc file.

Another issue I faced was the number and graph explanation.  It would be nice if you could add some explanation so that people can interpret the numbers/graph easily.

Thanks,
Ankit

**Comment by andre-richter** [comment]
Hi,

thanks for the feedback. I think I last tested this on 12.04, hence the missing vmalloc. I added it now in 16bcc6da8fdaa0192d088c16bda9c938dc27f16d.

Regarding the graph, it is just the standard probability histogram that you can create with matplotlib (https://matplotlib.org/1.2.1/examples/pylab_examples/histogram_demo.html). In this case, it shows the distribution of read latencies to the respective PCIe device (for all values within 3-sigma).

BR,
Andre

**Comment by andre-richter** [comment]
@ankitbhrdwj are you unblocked now?

Can we close this issue?

BR,
Andre

**Comment by ghost** [comment]
@andre-richter Yes, please. 
Thanks.
